src/app.js

```diff
@@ -573,7 +575,9 @@ var run = function () {
     compileTimeout = window.setTimeout(runCompiler, 300)
   }
 
-  editor.onChangeSetup(editorOnChange)
+  editor.event.register('changed', editorOnChange)
```

Comment:
I'm wondering whether we should rename the two events to `contentChanged` and `fileSwitched` to make a better distinction between the two events.